interfaces/apparmor/template_vars.go

```diff
@@ -35,6 +36,11 @@ func templateVariables(info *snap.Info, securityTag string) string {
 	fmt.Fprintf(&buf, "@{SNAP_REVISION}=\"%s\"\n", info.Revision)
 	fmt.Fprintf(&buf, "@{PROFILE_DBUS}=\"%s\"\n",
 		dbus.SafePath(securityTag))
-	fmt.Fprintf(&buf, "@{INSTALL_DIR}=\"/snap\"")
+
+	if dirs.SnapMountDir == "/snap" {
```

Comment:
Perhaps it would be easier to just always generate the 2nd form as it is not an issue in any way and there's less chance of breaking something in other setups.

Reply:
I changed this.